docs/client.rst

```diff
@@ -404,13 +404,21 @@ If you need to setup custom ssl parameters (use own certification
 files for example) you can create a :class:`ssl.SSLContext` instance and
 pass it into the connector::
 
-  >>> sslcontext = ssl.SSLContext(ssl.PROTOCOL_SSLv23)
-  >>> sslcontext.verify_mode = ssl.CERT_REQUIRED
-  >>> sslcontext.load_verify_locations("/etc/ssl/certs/ca-bundle.crt")
+  >>> sslcontext = ssl.create_default_context(cafile='/path/to/ca-bundle.crt')
   >>> conn = aiohttp.TCPConnector(ssl_context=sslcontext)
   >>> r = yield from aiohttp.request(
   ...     'get', 'https://example.com', connector=conn)
 
+You may also verify certificates via fingerprint::
+
+  >>> fp = '...'  # hex str of md5, sha1, or sha256 of expected cert (in DER)
+  >>> conn = aiohttp.TCPConnector(verify_fingerprint=fp)
+  >>> r = yield from aiohttp.request(
```

Comment:
I would to encourage `aiohttp.ClientSession` usage instead of `aiohttp.request`. Old documentation examples will be changed eventually, new ones should definitely use `ClientSession`.